4.cellprofiler_analysis/pipelines/analysis_4ch_with_sam.cppipe

```diff
@@ -211,40 +166,27 @@ MeasureColocalization:[module_num:10|svn_version:'Unknown'|variable_revision_num
+    Select images to measure:CL_488-1, CL_488_2, CL_561, DNA
     Measure within objects?:Yes
     Select objects to measure:Cells, Cytoplasm, Nuclei
     Subsampling factor for granularity measurements:0.25
     Subsampling factor for background reduction:0.25
     Radius of structuring element:10
     Range of the granular spectrum:16
 
-MeasureObjectIntensity:[module_num:12|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['Measures intensity within objects.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Select images to measure:488_1, 488_2, 561, DNA
+MeasureObjectIntensity:[module_num:11|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['Measures intensity within objects.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:CL_488-1, CL_488_2, CL_561, DNA
     Select objects to measure:Cells, Cytoplasm, Nuclei
 
-MeasureImageIntensity:[module_num:13|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['Measures whole image intensity not aggregated within objects.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Select images to measure:488_1, 488_2, 561, DNA
+MeasureImageIntensity:[module_num:12|svn_version:'Unknown'|variable_revision_number:4|show_window:False|notes:['Measures whole image intensity not aggregated within objects.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+    Select images to measure:CL_488-1, CL_488_2, CL_561, DNA
     Measure the intensity only from areas enclosed by objects?:No
     Select input object sets:Cells, Cytoplasm, Nuclei
     Calculate custom percentiles:No
     Specify percentiles to measure:10,90
 
-MeasureObjectNeighbors:[module_num:14|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['Measure nuclei neighbors that are adjacent to each other.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
-    Select objects to measure:Nuclei
-    Select neighboring objects to measure:Nuclei
-    Method to determine neighbors:Adjacent
-    Neighbor distance:50
-    Consider objects discarded for touching image border?:Yes
-    Retain the image of objects colored by numbers of neighbors?:No
-    Name the output image:ObjectNeighborCount
-    Select colormap:Blues
-    Retain the image of objects colored by percent of touching pixels?:No
-    Name the output image:PercentTouching
-    Select colormap:Oranges
-
-MeasureObjectNeighbors:[module_num:15|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['Measure cell neighbors that are adjacent to each other.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+MeasureObjectNeighbors:[module_num:13|svn_version:'Unknown'|variable_revision_number:3|show_window:False|notes:['Measure cell neighbors that are adjacent to each other.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Is there a reason you only want to measure cell neighbors and not nuclei neighbors as well?

Reply:
Nope, no reason. Adding now!